code/_onclick/click.dm

```diff
@@ -1,145 +1,231 @@
-/*
-	client/Click is called every time a client clicks anywhere, it should never be overridden.
+// 1 decisecond click delay (above and beyond mob/next_move)
+//This is mainly modified by click code, to modify click delays elsewhere, use next_move and changeNext_move()
+/mob/var/next_click	= 0
+
+
+// THESE DO NOT EFFECT THE BASE 1 DECISECOND DELAY OF NEXT_CLICK
+/mob/var/next_move_adjust = 0 //Amount to adjust action/click delays by, + or -
+/mob/var/next_move_modifier = 1 //Value to multiply action/click delays by
```

Comment:
These variables should be defined under `mob_defines.dm`

Reply:
This is /tg/ though